### ISSUE TITLE
Check that yeast lncRNA type is correct

### ISSUE THREAD

**Issue by AntonPetrov** [RNAcentral/rnacentral-webcode#179]
Currently there is just 1 lncRNA from _S pombe_: http://rnacentral.org/rna/URS000074AB82/4896 but there should be more. Are the RNA types chosen accurately for yeast?

PomBase does not seem to have lncRNA as a non-coding gene type, but there should be more lncRNAs. Here is a [relevant review](http://www.sciencedirect.com/science/article/pii/S1874939915001716).

**Comment by blakesweeney** [comment]
I've looked into this a bit. In terms of the given data this is 'correct', in that we only have sequence which contains an xref that assigns an `lncRNA` rna_type. This annotation does not come from Pombase. Thus we only have 1 lncRNA in pombe. So at least this isn't a bug somewhere. For reference I downloaded the genome coordinates for release 6 and see that there are no sequences called lncRNA. 

I read over the paper and for pombe it discusses meiRNA quite a bit. This is the only sequence we have annotated as lncRNA. So at least our annotation there is good. 

However the review does discuss other RNA's like prt (Pho1-Repressing Transcript). I can't find this easily in RNAcentral. I found it in pombase by finding the gene it is downstream of then looking around. The RNA (http://rnacentral.org/rna/URS00003F9306/4896) that is likely this gene is marked as `misc_RNA` and called predicted. I've looked at another example and I see it is marked as `antisense`. 

Overall, I think this is an annotation problem with what we are given. We may want to ask Pombase to update their annotations, to include the genes from this paper along with improving their annotation of lncRNA. I'm not sure how much work it will be to track these sequences down though.  Altering our import procedures to always treat large misc_RNA or antisense as lncRNA is an option as well. 

I think using `misc_RNA` and `antisense` instead of `lncRNA` is a common issue with the data we are given, but I'm not sure of a safe way to fix it. Converting them all to lncRNA is probably not safe. Once we move to SO terms, I'm ok with treating `antisense` as a qualifier of other annotations and suggesting to the user to also use `antisense` when searching for lncRNA's. Until then I'm not certain of how to fix this.
